src/System.Security.Cryptography.Encoding/src/Internal/Cryptography/OidLookup.OSX.cs

```diff
@@ -53,7 +52,7 @@ private static string NativeFriendlyNameToOid(string friendlyName, OidGroup oidG
         // and OpenSSL produce different answers.  Since the answers originally came from OpenSSL
         // on macOS, this preserves the OpenSSL names.
         private static readonly Dictionary<string, string> s_extraFriendlyNameToOid =
-            new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
+            new Dictionary<string, string>(8, StringComparer.OrdinalIgnoreCase)
```

Comment:
Consider making the 8 a constant and adding a #if DEBUG cctor that asserts that the count is correct (both here and the other one)

Reply:
Ok